webhook/hooks.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "deploy",
+    "execute-command": "/home/customdomain/deploy.sh",
+    "command-working-directory": "/home/customdomain",
+    "pass-environment-to-command": [
+      {
+        "source": "payload",
+        "name": "tag",
+        "envname": "TAG"
+      }
+    ],
+    "trigger-rule": {
+      "match": {
+        "type": "payload-hmac-sha256",
+        "secret": "TODO: update me to match secret stored in GitHub Actions...",
```

Comment:
Wanted to confirm, Is this secret supposed to be a path to a secret file or something like that? Or is this going to be manually updated later (and not comnitted to git)? Doesn't seems like a secret if we commit this here

Reply:
good point

Reply:
So, will you ssh into the server to change this manually?